### ISSUE TITLE
Remove useless outer parens

### ISSUE THREAD

**Issue by impredicative** [cube2222/jql#3]
Consider removing the redundant outer parens since all your examples have it. Why can't it just be imputed? Why do I have to remember to type two extra characters that convey zero extra information?

**Comment by cube2222** [comment]
You mean trailing parentheses of the whole query? That's actually a great idea! I'll think about how to include this into the grammar.

**Comment by impredicative** [comment]
Yes, the leading and trailing paren of the whole query. I would make it optional, such that the query continues to work either way, with or without the outer parens, assuming this makes sense.

**Comment by cube2222** [comment]
There will only ever be a single leading one, but trailing ones are definitely not useful, and were annoying to me too.

Thanks for the issue!

**Comment by impredicative** [comment]
I would remove the requirement of the single leading paren, making it optional. I figure that the program should be able to impute it since all the examples have it. If something is always present, it conveys no information.
